### DISCUSSION TITLE
Windows build - Stotiks

### DISCUSSION THREAD

**Discussion by gilmotta** [madMAx43v3r/chia-plotter#344] [General]
Hello All,

I'm trying to build the Windows port from Stotiks but VS is giving me errors saying that the CmakeList is missing under the Libs.

![image](https://user-images.githubusercontent.com/4481889/121781858-b946c580-cb74-11eb-8322-abf10cb2f1f0.png)

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		CMake Error at C:\python_dev\development_chia\Stotiks_Windows\chia-plotter\CMakeLists.txt:5 (add_subdirectory):
  The source directory

    C:/python_dev/development_chia/Stotiks_Windows/chia-plotter/lib/bls-signatures

  does not contain a CMakeLists.txt file.	chia	C:\python_dev\development_chia\Stotiks_Windows\chia-plotter\CMakeLists.txt	5
```	

Stotiks or anyone, can you please provide instructions to build for Windows? I appreciate it.

Here is the link to the port:
https://github.com/stotiks/chia-plotter

**Comment by XIII39** [comment]
In Linux we need to grab the submodule libs before you compile with cmake - `git submodule update --init`
This gets the bls-signatures and BLAKE3 files.

Perhaps we need to do something similar in Windows?

**Reply by SebMoore** [reply]
> In Linux we need to grab the submodule libs before you compile with cmake - `git submodule update --init`
> This gets the bls-signatures and BLAKE3 files.
> 
> Perhaps we need to do something similar in Windows?

@gilmotta I believe this is the correct solution - git cloning the plotter repo will not also clone the Blake3 and bls-signatures library - you will need to manually download them, or use `git submodule update --init`, or use one of the many git submodule extensions for VS.
